charts/rancher-vsphere-csi/templates/node/windows-daemonset.yaml

```diff
@@ -99,11 +99,6 @@ spec:
                   fieldPath: metadata.namespace
             - name: NODEGETINFO_WATCH_TIMEOUT_MINUTES
               value: "1"
-          securityContext:
-            privileged: true
-            capabilities:
-              add: ["SYS_ADMIN"]
-            allowPrivilegeEscalation: true
```

Comment:
Did you intentionally remove this block? If you did, can you comment out the block and add a comment like `# Rancher specific: <reason-why-it-was-removed>`? This will prevent the next person who updates the chart from bringing this back into the chart since I see this block upstream in tag v2.5.0.

Reply:
@PennyScissors this is the windows daemonset and they accidentally had it added in 2.4.1, but it was fixed upstream. Here is the diff.

![image](https://user-images.githubusercontent.com/1568133/159378719-1239c08f-ef7d-451e-9233-85467eb4f3f6.png)